### ISSUE TITLE
Zarf Skeleton Doesn't load files outside of directory

### ISSUE THREAD

**Issue by runyontr** [defenseunicorns/zarf#1771]
### Environment
Device and OS: linux
App version: main branch
Kubernetes distro being used: n/a
Other:

### Steps to reproduce
1. Published [this package](https://github.com/defenseunicorns/zarf-package-big-bang/tree/main/k3d) as a skeleton:

```shell
 zarf-package-k3d git:(main) ✗ zarf package publish .  oci://ghcr.io/runyontr 

Using config file /home/tom/Workspaces/zarf-package-k3d/zarf-config.yaml

Saving log file to /tmp/zarf-2023-06-02-05-42-58-3344554324.log

                                                                                       
  📦 K3D COMPONENT                                                                     
                                                                                       


                                                                                       
  📦 INIT COMPONENT                                                                    
                                                                                       


                                                                                       
  📦 METALLB COMPONENT                                                                 
                                                                                       

  ✔  Loading 1 K8s manifests                                                                                                           
  ✔  All of the checksums matched!

                                                                                       
  📦 PACKAGE PUBLISH k3d-local:v1.26.0-skeleton                                        
                                                                                       

  •  Publishing package to ghcr.io/runyontr/k3d-local:v1.26.0-skeleton
  ✔  Prepared 5 layers                                                                                                                 
  ✔  f1300d0e063c [application/vnd.unknown.config.v1+json]                                                                             
  ✔  f596441c2a26 checksums.txt                                                                                                        
  ✔  0913ba1c0ad9 components/k3d.tar                                                                                                   
  ✔  f7fc830805c8 components/metallb.tar                                                                                               
  ✔  6e6aca467266 zarf.yaml                                                                                                            
  ✔  85d16f549a75 components/init.tar                                                                                                  
  ✔  01838056b4b5 [application/vnd.oci.image.manifest.v1+json]                                                                         
  ✔  Published ghcr.io/runyontr/k3d-local:v1.26.0-skeleton [application/vnd.oci.image.manifest.v1+json]                                

  •  Example of importing components from this package:


- name: import-k3d
  import:
    name: k3d
    url: oci://ghcr.io/runyontr/k3d-local:v1.26.0-skeleton
- name: import-init
  import:
    name: init
    url: oci://ghcr.io/runyontr/k3d-local:v1.26.0-skeleton
- name: import-metallb
  import:
    name: metallb
    url: oci://ghcr.io/runyontr/k3d-local:v1.26.0-skeleton

```


And then consume it with a zarf package like this one:  https://github.com/runyontr/uds-core-k3d/blob/main/zarf.yaml#L33-L36


and get the following error:

```shell
$ zarf package create .
....

                                ######  ###  #####     ######     #    #     #  #####  
                                #     #  #  #     #    #     #   # #   ##    # #     # 
                                #     #  #  #          #     #  #   #  # #   # #       
                                ######   #  #  ####    ######  #     # #  #  # #  #### 
                                #     #  #  #     #    #     # ####### #   # # #     # 
                                #     #  #  #     #    #     # #     # #    ## #     # 
                                ######  ###  #####     ######  #     # #     #  #####  

  ✔  Processing helm chart bigbang                                                                                                                                                                                                                                                             
  ⠋  Templating helm chart bigbang (0s)                                                                                                                                                                                                                                                        
     ERROR:  Failed to create package: unable to process extensions: unable to process bigbang extension: unable
             to template Big Bang Chart: unable to load chart data: unable to parse chart values: open
             ../values/values.yaml: no such file or directory
```

### Expected result

The skeleton package would bring in everything it references

### Actual Result

It just pulls in the folder provided.

### Visual Proof (screenshots, videos, text, etc)

### Severity/Priority

### Additional Context
Add any other context or screenshots about the technical debt here.

**Comment by Noxsios** [comment]
@runyontr Is this still an issue after [0.27.1](https://github.com/defenseunicorns/zarf/releases/tag/v0.27.1)?

**Comment by runyontr** [comment]
This looks fixed in 0.27.1.  Thanks @Noxsios !
